assets/js/blocks/collection-filters/inner-blocks/stock-filter/frontend.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import {
+	store as interactivityStore,
+	navigate,
+} from '@woocommerce/interactivity';
+
+/**
+ * Internal dependencies
+ */
+import { HTMLElementEvent } from '../../../../types';
+import { DropdownContext } from '../../../../../../packages/interactivity-components/dropdown';
```

Comment:
We can use some of our [aliases](https://github.com/woocommerce/woocommerce-blocks/blob/c84ca9fe5a7bd1cd11dfa69129c8679f2e1bb777/tsconfig.base.json#L25-L71) here.

Reply:
[Here](https://github.com/woocommerce/woocommerce-blocks/pull/11663/commits/911a0005ed70a3788e575e03f04a1550c4fb7b6b) I added a new alias for the interactivity components (its a new path I added for this PR). I used the existing alias for the types.